packages/starlight/utils/createTranslationSystem.ts

```diff
@@ -42,7 +42,7 @@ export function createTranslationSystem<T extends i18nSchemaOutput>(
 	}
 
 	const i18n = i18next.createInstance();
-	i18n.init({
+	void i18n.init({
```

Comment:
Is there a rule requiring `void` for any side-effect function call like this? Seems a bit unnecessary unless there’s an argument for it I’m missing?

(In the past I’ve usually only ever used `void` in scenarios like `() => void fn()` to shield the outer function from returning the inner function call’s returned value.)

Reply:
This one I need to investigate a bit more, but with `typescript-eslint`, the `void` operator is only used here to explicitly mark a Promise as intentionally not awaited. This is the case here as we are explicitly passing it the `resources` to initialize with and not relying on any loading from the library, which also allows us to keep the entire chain synchronous and avoid having to switch everywhere to `await useTranslations()` for example.

Depending on where we land at the end, this part will probably need to be updated, at least to comment it, or switch to a different approach.